### ISSUE TITLE
@import '_file.scss'; not working

### ISSUE THREAD

**Issue by aivascu** [jasonsanjose/brackets-sass#79]
Might be a noobish misunderstanding, but importing a dependency file doesn't work for me.
When trying to import a partial scss file I'm getting the 'file to import not found or unreadable' error.
The main scss file and the partial file are in the same directory.

**Comment by jasonsanjose** [comment]
No need to add the underscore or the file extension for partials. Try `@import "file"`.

**Comment by aivascu** [comment]
Thanks Jason!
It still gives me the error but at least it generates the *.css file correctly.
Also in live preview the changes in the imported *.scss file don't apply until the importing file is changed(re-saved).

**Comment by Belkar** [comment]
>Also in live preview the changes in the imported *.scss file don't apply until the importing file is changed(re-saved).
Same problem here. This makes developing and styling tedious

**Comment by jasonsanjose** [comment]
Could you try syncing to the latest https://github.com/jasonsanjose/bourbon-example. I updated it to include a local partial file for importing.

**Comment by jasonsanjose** [comment]
@Belkar also please try the latest 1.0.4-83 update. Are you on mac or win?

**Comment by aivascu** [comment]
Updated the extension, still getting the same behavior. 
Bourbon has the same behavior too.
I am running Windows 8.1.
![image](https://cloud.githubusercontent.com/assets/7030530/5832748/6de5dffc-a152-11e4-844d-c61849d2799f.png)

**Comment by jasonsanjose** [comment]
Hmm. I'm on windows and I'm still not seeing an issue. Have you tried quitting and restarting Brackets?

**Comment by Belkar** [comment]
>@Belkar also please try the latest 1.0.4-83 update. Are you on mac or win?

Thanks it works now :smile:

**Comment by aivascu** [comment]
@jasonsanjose yes I restarted Brackets several times. Haven't tried a reinstall though. That usually cuts it with programs on Windows.
Here is a demo of my problem.

![brackets](https://cloud.githubusercontent.com/assets/7030530/5833852/28101ade-a160-11e4-9ded-3f8388de8de0.gif)

@Belkar what OS are you running?

**Comment by Belkar** [comment]
I'm using Windows 8.1.

**Comment by aivascu** [comment]
So I tried several things. I uninstalled Brackets; removed the installation folder from Program Files; cleaned the registries, the temp files and the Roaming folder; Installed Brackets; Installed brackets-sass.
Not sure which one of this things helped but it works for me too. I'm still getting the false error but at least it updates the partial styles in live preview.
@Belkar it worked for me with the default config.

![brackets](https://cloud.githubusercontent.com/assets/7030530/5835019/38842264-a16c-11e4-91ee-6e17fe0e59b9.gif)

Thanks for the help guys.
